### PR TITLE
bitwarden_rs: 1.19.0 -> 1.20.0

### PR DESCRIPTION
**Pull request by Munksgaard**
###### Motivation for this change

https://github.com/dani-garcia/bitwarden_rs/releases/tag/1.20.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/bitwarden_rs/default.nix [hunk 1]
```diff
@@ -8,7 +8,7 @@ let
 
 in rustPlatform.buildRustPackage rec {
   pname = "bitwarden_rs";
-  version = "1.19.0";
+  version = "1.20.0";
 
   src = fetchFromGitHub {
     owner = "dani-garcia";
```
Comment (SuperSandro2000) [inline pkgs/tools/security/bitwarden_rs/default.nix:14]: You forgot to update the src hash.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117931` at cf653c15 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>bitwarden_rs (bitwarden_rs-sqlite)</li>
    <li>bitwarden_rs-mysql</li>
    <li>bitwarden_rs-postgresql</li>
  </ul>
</details>

File: pkgs/tools/security/bitwarden_rs/default.nix (hunk 1 above)
Comment (Munksgaard) [reply pkgs/tools/security/bitwarden_rs/default.nix:14]: Sorry, I've done that now.

Comment (SuperSandro2000) [reply pkgs/tools/security/bitwarden_rs/default.nix:14]: ofborg still fails. not sure why.

Comment (Munksgaard) [reply pkgs/tools/security/bitwarden_rs/default.nix:14]: The log mentions cargo256sha being out of date. I've tried to follow the steps suggested and pushed a new set of commits.

**Comment by zimbatm** [comment]
Result of `nixpkgs-review pr 117931` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package blacklisted:</summary>
  <ul>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>bitwarden_rs (bitwarden_rs-sqlite)</li>
    <li>bitwarden_rs-mysql</li>
    <li>bitwarden_rs-postgresql</li>
  </ul>
</details>

**Comment by Munksgaard** [comment]
I don't think the failing test has anything to do with bitwarden_rs? It's building firefox that's timing out, as far as I can tell.

**Comment by Munksgaard** [comment]
@SuperSandro2000 What do you think?

**Pull request merged**